.github/workflows/build-and-deploy.yml

```diff
@@ -204,6 +205,7 @@ jobs:
           echo "SIGNTOOL=git signtool" >>$GITHUB_ENV
 
       - name: Build ${{env.PACKAGE_TO_BUILD}}
+        timeout-minutes: 150
```

Comment:
This one needs a special case for LLVM.
 something like `timeout-minutes ${{ env.PACKAGE_TO_BUILD== 'mingw-w64-llvm' && '
300 || 150 }}`

Reply:
Done!